fusor-ember-cli/app/controllers/engine/discovered-host.js

```diff
@@ -14,12 +14,12 @@ export default Ember.Controller.extend({
   allDiscoveredHosts: [],
 
   engineNextRouteName: function() {
-    if (this.get('model.rhev_is_self_hosted')) {
+    if (this.get('rhevIsSelfHosted')) {
```

Comment:
@knowncitizen, notice that on line 9
+  rhevIsSelfHosted: Ember.computed.alias("controllers.deployment.model.rhev_is_self_hosted"),

so my thought is to make all controller level attribute camelCase and all model level attributes snake_case. Then, it will be easier to differentiate. In the example, it was a bug since `rhev_is_self_hosted` is not an attribute of model for `engine/discovered-host`